### PR TITLE
LPS-106344 | Create Experiences

### PR DESCRIPTION
**Pull request by andresfulla**
@ealonso @dgarciasarai

### CODE REVIEW

**Comment by andresfulla** [comment]
Review only `LPS-106344` please

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js [hunk 1]
```diff
@@ -24,6 +24,25 @@ import {
 } from './actions';
 import ExperienceToolbarSection from './components/ExperienceToolbarSection';
 
+/**
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js]: Check if there's any other underscore. Avoid!!!

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js [hunk 2]
```diff
@@ -94,6 +145,21 @@ function createExperienceReducer(state, payload) {
 
 	// TODO update layoutData
 
+	nextState = _updateFragmentEntryLinksEditableValues(
+		nextState,
+		fragmentEntryLinks
+	);
+
+	nextState = _setExperienceLock(nextState, {
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js]: check if this should be true

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js [hunk 3]
```diff
@@ -94,6 +145,21 @@ function createExperienceReducer(state, payload) {
...
+		hasLockedSegmentsExperiment: true
+	});
+
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/index.js]: review this TODOs

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/reducers/utils.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+function switchLayoutData(state, {currentExperienceId, targetExperienceId}) {
+	let nextState = state;
+	const {layoutData: prevLayoutData, layoutDataList} = nextState;
+
+	const layoutDataItem = state.layoutDataList.find(
+		layoutDataItem =>
+			layoutDataItem.segmentsExperienceId === targetExperienceId
+	);
+
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/reducers/utils.js]: This try is temporary until the backend catches up with the layoutData shape.

**Comment by julien** [comment]
> Reviewed with @julien

Yes, that's true

**Comment by andresfulla** [comment]
Sent! https://github.com/brianchandotcom/liferay-portal/pull/83035

**Pull request closed without merging**